src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements Writer {
+    @Override
+    public void writeToFile(String resultingReport, String path) {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(path));
+            writer.write(resultingReport);
+            writer.close();
```

Comment:
If you switch to a try-with-resources statement, you can remove this explicit call to `writer.close()`, as it will be handled automatically.